package.json

```diff
@@ -98,9 +99,9 @@
         "@nrwl/nx-cloud": "14.5.1",
         "@suite-common/wallet-types": "*",
         "@types/jest": "^26.0.24",
-        "@types/node": "^16.11.49",
-        "@types/prettier": "^2.7.0",
-        "@typescript-eslint/eslint-plugin": "^5.33.1",
+        "@types/node": "^18.11.9",
```

Comment:
@Nodonisko Is it okay to update to `@types/node` v18 even though we use node 16?